rslib/src/error/mod.rs

```diff
@@ -12,15 +12,17 @@ pub use db::{DbError, DbErrorKind};
 pub use filtered::{CustomStudyError, FilteredDeckError};
 pub use network::{NetworkError, NetworkErrorKind, SyncError, SyncErrorKind};
 pub use search::{ParseError, SearchErrorKind};
+use snafu::{Backtrace, ErrorCompat, Snafu};
 use tempfile::PathPersistError;
 
 use crate::{i18n::I18n, links::HelpPage};
 
 pub type Result<T, E = AnkiError> = std::result::Result<T, E>;
 
-#[derive(Debug, PartialEq)]
```

Comment:
After you added the manual PartialEq implementations, this can be derived again, so presumably those equality changes are no longer required.

Reply:
Yeah, I removed the equality checks afterwards to see if it was practical.